run_cumulus_task.py

```diff
@@ -19,12 +28,24 @@ def run_cumulus_task(task_function, cumulus_message, context):
 
     message_adapter_disabled = os.environ.get('CUMULUS_MESSAGE_ADAPTER_DISABLED')
 
-    if message_adapter_disabled:
-        return task_function(cumulus_message, context)
+    if message_adapter_disabled is True:
+        try:
+            return task_function(cumulus_message, context)
+        except Exception as exception:
```

Comment:
This should be `except WorkflowError as exception:`

Reply:
I'm not sure. Will it definitely be a `WorkflowError` exception or do we need to check for an exception where the first arg is a string with `WorkflowError`?

Should this package expose the `WorkflowError` class for task authors to raise in their tasks?

Reply:
Well, the way that the original JS sled was implemented, it looked for an exception whose name contained "WorkflowError".

Reply:
ok, that has me leaning toward having a task author using something like `raise Exception('SomeWorkflowError')`

then this will check using:

```py
except Exception as exception:
    name = exception.args[0]
    if ('WorkflowError' in name):                
        cumulus_message['payload'] = {}
        cumulus_message['exception'] = name
        return cumulus_message
    else:
        raise exception 
```

And that'll mirror really closely what's happening now in cumulus-message-adapter-js